docs/sources/awscodecommit.md

```diff
@@ -3,11 +3,11 @@
 This event source captures notifications from an [AWS CodeCommit repository][cc-docs] whenever a specific action, such
 as a new commit or the creation of a pull request, happens in this repository.
 
-## Prerequisites
+## Prerequisite(s)
 
-* CodeCommit Repository and Branch
-* Amazon Resource Name (ARN)
-* API Credentials
+- CodeCommit Repository and Branch
+- Amazon Resource Name (ARN)
+- API Credentials
```

Comment:
Not saying we shouldn't do this, but both `*` and `-` are perfectly valid and represent the same thing in Markdown.

Most pages in our docs, including index pages, are currently using `*`. With this change, we end up with a mix of `*` and `-` within the _same_ page, which is quite inconsistent and maybe not really justified.

@daceynolan any thoughts?

Reply:
@antoineco thanks for that feedback. That is my fault, my template shows `-` instead of `*`. I never have used `*` before so that was something I just learned. 

If we have more `*` let's keep it like that. 

Thomas, do you mind changing these back to `*` and do you mind changing it in the template?